packages/experimental/Spinner/src/Spinner.styling.win32.ts

```diff
@@ -1,47 +1,16 @@
-import type { UseStylingOptions, Variant } from '@fluentui-react-native/framework';
+import type { UseStylingOptions } from '@fluentui-react-native/framework';
 import { buildProps } from '@fluentui-react-native/framework';
 
 import type { SpinnerProps, SpinnerSlotProps, SpinnerTokens } from './Spinner.types.win32';
 import { spinnerName } from './Spinner.types.win32';
-import { defaultSpinnerTokens } from './SpinnerTokens';
-
-export const diameterSizeMap: { [key: string]: number } = {
-  tiny: 20,
-  'x-small': 24,
-  small: 28,
-  medium: 32,
-  large: 36,
-  'x-large': 40,
-  huge: 44,
-};
-export const lineThicknessSizeMap: { [key: string]: number } = {
-  tiny: 2,
-  'x-small': 2,
-  small: 2,
-  medium: 3,
-  large: 3,
-  'x-large': 3,
-  huge: 4,
-};
-
-export const textStyleMap: { [key: string]: Variant } = {
-  tiny: 'body1',
-  'x-small': 'body1',
-  small: 'body1',
-  medium: 'subtitle2',
-  large: 'subtitle2',
-  'x-large': 'subtitle2',
-  huge: 'subtitle1',
-};
+import { defaultSpinnerTokens, diameterSizeMap } from './SpinnerTokens';
 
 export const stylingSettings: UseStylingOptions<SpinnerProps, SpinnerSlotProps, SpinnerTokens> = {
   tokens: [defaultSpinnerTokens, spinnerName],
   tokensThatAreAlsoProps: 'all',
   slotProps: {
     root: buildProps(
       (tokens: SpinnerTokens) => ({
-        accessibilityRole: 'progressbar',
-        accessible: true,
         style: {
           width: diameterSizeMap[tokens.size],
```

Comment:
For other controls we've set the width token to the output of diameterSizeMap in the Tokens file and then accessed tokens.width here. This allows the user to specify a particular width if they want to. Can we follow that pattern?

Reply:
So I'm a little confused on how I would do this. Do you mean that I define width and height tokens under SpinnerTokens, and set their default values based on size in the Tokens file? But you mentioned that size is not usually set as component tokens themselves, so size isn't accessible in the tokens file

Reply:
So the pattern for other components has been to set size as a state for a component, which we would then set defaults for in the tokens file. For example state would be something like "small, medium, large", and then in the tokens file it would do something like:

"small": {
    width: foo1, height: foo2,
},
"medium" :{
    width: foo3, height: foo4,
}

etc.

Reply:
Badge and Avatar should have examples.

Reply:
Gotcha! Thanks :)